### ISSUE TITLE
Unable to tap on a series or an author with Voiceover enabled

### ISSUE THREAD

**Issue by mohammad-aloufi** [advplyr/audiobookshelf-app#497]
This is an accessibility issue. I'm unable to tap on the author name to bring their books, or a book to bring up its contents.
Double tapping with Voiceover enabled is the same as tapping on an item once with Voiceover disabled.
I'm on iOS16.2, with abs version 0.9.60

Voiceover is the built-in screen reader that apple provides for all devices so people who are either totally blind or low vision can read the screen.

**Comment by hobesman** [comment]
To clarify, you're unable to use voice over to tap on those items? Or are you saying the buttons can't be (double) tapped with a finger if voiceover is enabled?

**Comment by mohammad-aloufi** [comment]
I can't double tap on them to open that series or author with Voiceover enabled
